R/pkg/R/DataFrame.R

```diff
@@ -1268,6 +1268,82 @@ setMethod("dapplyCollect",
             ldf
           })
 
+#' gapply
+#'
+#' Apply a R function to each group of a DataFrame. The group is defined by an input
+#' grouping column.
+#'
+#' @param x A SparkDataFrame
+#' @param func A function to be applied to each group partition specified by grouping
```

Comment:
Minor comment: It would be good to say what the function will get as its input. Right now its the key and a dataframe with the grouping columns ?